### PR TITLE
Add RubyMine icon

### PR DESCRIPTION
**Pull request by davidlj95**
image

**Issue:** #10706 

**Popularity metric:**

[Google Trends comparison of other JetBrains IDEs already included: CLion, PhpStorm, Resharper & Rider](https://trends.google.com/trends/explore?geo=US&q=CLion,phpstorm,rubymine,resharper,jetbrains%20rider). Not the coolest kid on block, but still there 😛 

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Using same approach as CLion, WebStorm, ... and other similar JetBrains' IDEs whose logo is similar too

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -13204,6 +13204,12 @@
             "hex": "E9573F",
             "source": "https://rubygems.org/pages/about"
         },
+        {
+            "title": "RubyMine",
+            "hex": "000000",
+            "source": "https://www.jetbrains.com/company/brand/logos/",
+            "guidelines": "https://www.jetbrains.com/company/brand/"
```
Comment (adamrusted) [inline _data/simple-icons.json:13211]:
```suggestion
            "source": "https://www.jetbrains.com/company/brand/logos",
            "guidelines": "https://www.jetbrains.com/company/brand"
```

File: icons/rubymine.svg
Comment (adamrusted) [inline icons/rubymine.svg:1]: Using the RubyMine icon found at source (in red) there are some quite significant differences. Would you mind giving this another go please @davidlj95?
image

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (davidlj95) [reply _data/simple-icons.json]: Done in 5cc2150faff1c411341d691535fba548ccfb7b0c

For consistency. Though being nitty 🧐 canonical URLs are the ones with trailing slash

The guidelines `/brand/` one has trailing slash as canonical URL
```html
<link rel="canonical" href="https://www.jetbrains.com/company/brand/">
```
And if accessing the non-trailing slash, you get a redirect to the canonical one:

image

Same happens with the source `brand/logos` URL:

image

`/brand/logos/` then redirects to `/brand/#logos-and-icons-jetbrains-logos`

image

If have some time will open a PR normalizing all JetBrains URLs to save some GET requests around

File: icons/rubymine.svg
Comment (davidlj95) [reply icons/rubymine.svg:1]: Interesting! Let me give it another try, thanks 😊

File: _data/simple-icons.json (hunk 1 above)
Comment (adamrusted) [reply _data/simple-icons.json]: We try to keep all urls without the trailing slash - it saves a few bytes per icon, and as our collection grows, every little helps!

Comment (davidlj95) [reply _data/simple-icons.json]: Sure, let's save some bytes then 👍 Not a big deal anyway as long the link works

File: icons/rubymine.svg
Comment (davidlj95) [reply icons/rubymine.svg:1]: Done in 70265c7b4a5322b3ec20040a3050b1d73d02b6e4

Seems they updated it quite recently:

image

Now the `M` touches the baseline

**Review by adamrusted — APPROVED** [review]
Perfect match, thanks for your work on this @davidlj95! 🎉

**Comment by adamrusted** [comment]
Don't worry about the newline - I'll deal with it on my end :)

**Comment by davidlj95** [comment]
> Don't worry about the newline - I'll deal with it on my end :)

Whooops! Just noticed that and the title missing. Fixed it in last push

**Pull request merged**